packages/typing-extensions/meta.yaml

```diff
@@ -1,9 +1,18 @@
 package:
   name: typing-extensions
-  version: 3.10.0.0
+  version: 4.0.1
+  # I needed some dumb hacks to get this to work. TODO: fix build system after
+  # 0.19 release.
```

Comment:
Maybe let's open an issue to remember it?

Reply:
Sure. My thought is that with #2027, we can switch to using unmodified wheels from PyPI for all of our unpatched pure python packages.

Reply:
Yes, that would be ideal. Though we still might want to unvendor tests which would require repacking of the wheel but shouldn't be too hard (and won't need to run setup.py).

Reply:
Yeah, my setup in #2027 divides up the process of creating the package into:
1. create or acquire wheel
2. `python -m wheel unpack` the wheel
3. Apply post script and unvendor tests
4. `python -m wheel pack`